django_project/realtime/tasks/earthquake.py

```diff
@@ -104,67 +106,6 @@ def retrieve_felt_earthquake_list():
         LOGGER.debug('Failed to connect to {url}'.format(url=target_url))
 
 
-@app.task(queue='inasafe-django')
```

Comment:
Deleted this whole blob of unoptimized pizzas that will make your computer freezes to death when debugging.